### PR TITLE
Fixed pool/virtual-server resource default description issue

### PR DESCRIPTION
**Pull request by RavinderReddyF5**

### CODE REVIEW

File: bigip/resource_bigip_ltm_pool.go [hunk 1]
```diff
@@ -187,13 +186,15 @@ func resourceBigipLtmPoolUpdate(d *schema.ResourceData, meta interface{}) error
...
 		AllowSNAT:         d.Get("allow_snat").(string),
 		LoadBalancingMode: d.Get("load_balancing_mode").(string),
-		Description:       d.Get("description").(string),
 		SlowRampTime:      d.Get("slow_ramp_time").(int),
 		ServiceDownAction: d.Get("service_down_action").(string),
 		ReselectTries:     d.Get("reselect_tries").(int),
 		Monitor:           strings.Join(monitors, " and "),
 	}
 
+	if d.Get("description").(string) != "" {
```
Comment (dannyk81) [inline bigip/resource_bigip_ltm_pool.go]: I don't think you need this conditional, if the bigip returns an empty string, that's what we should store in the state.

File: bigip/resource_bigip_ltm_virtual_server.go [hunk 2]
```diff
@@ -401,7 +399,9 @@ func resourceBigipLtmVirtualServerUpdate(d *schema.ResourceData, meta interface{
 		TranslateAddress: d.Get("translate_address").(string),
 		VlansEnabled:     d.Get("vlans_enabled").(bool),
 	}
-
+	if d.Get("description").(string) != "" {
```
Comment (dannyk81) [inline bigip/resource_bigip_ltm_virtual_server.go]: same here, this conditional is not needed

File: bigip/resource_bigip_ltm_pool.go (hunk 1 above)
Comment (RavinderReddyF5) [reply bigip/resource_bigip_ltm_pool.go]: @dannyk81 
Updated with comments and submitted change

File: bigip/resource_bigip_ltm_virtual_server.go (hunk 2 above)
Comment (RavinderReddyF5) [reply bigip/resource_bigip_ltm_virtual_server.go]: @dannyk81 
Updated with comments and submitted change

**Pull request merged**